yarn.lock

```diff
@@ -24297,7 +24308,7 @@ require-from-string@^2.0.1:
   resolved "https://registry.yarnpkg.com/require-from-string/-/require-from-string-2.0.2.tgz#89a7fdd938261267318eafe14f9c32e598c36909"
   integrity sha512-Xf0nWe6RseziFMu+Ap9biiUbmplq6S9/p+7w7YXP/JBHhrUDDUhwa+vANyubuqfZWTveU//DYVGsDG7RKL/vEw==
 
-require-in-the-middle@^5.0.2, require-in-the-middle@^5.0.3:
+require-in-the-middle@^5.0.3:
```

Comment:
Also seems like we can remove this since we have `require-in-the-middle@^5.1.0` below